cocos/rendering/custom/layout-graph.ts

```diff
@@ -471,11 +473,101 @@ export class DescriptorSetData {
     declare /*refcount*/ descriptorSet: DescriptorSet | null;
 }
 
+export class DescriptorGroupBlockData {
```

Comment:
Is it needed to be exported to developer?

Reply:
I will refactor the types and move them into internal modules.

Reply:
> I will refactor the types and move them into internal modules.

That will be good, in that case, all properties could be mangled by adding `/** @mangle */` for the classes or properties.

Reply:
> I will refactor the types and move them into internal modules.

Will you do it in this PR or other PR?